R/stan_betareg.fit.R

```diff
@@ -20,7 +20,7 @@
 
 stan_betareg.fit <- function (x, y, z = NULL, weights = rep(1, NROW(x)), offset = rep(0, NROW(x)),
                               link = c("logit", "probit", "cloglog", "cauchit", "log", "loglog"), 
-                              link.phi = c("log", "identity", "sqrt"), ...,
+                              link.phi = "log", ...,
```

Comment:
Now that I look more closely at the betareg package, it looks like `stan_betareg` doesn't match`betareg` in terms of the default settings of `link.phi`. Is that intentional? I see these defaults for `link.phi`:

* `betareg`: `link.phi = NULL`
* `stan_betareg`: `link.phi = c("log", "identity", "sqrt")`